src/org/traccar/database/QueryBuilder.java

```diff
@@ -185,6 +185,23 @@ public QueryBuilder setLong(String name, long value) throws SQLException {
         return this;
     }
 
+    public QueryBuilder setLong(String name, long value, boolean nullIfZero) throws SQLException {
```

Comment:
Please make `setLong` function with 2 parameters call this one to avoid duplication.